docs/eventing/brokers/broker-admin-config-options.md

```diff
@@ -68,47 +44,86 @@ data:
 
 ```yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: config-br-defaults
   namespace: knative-eventing
-  labels:
-    eventing.knative.dev/release: devel
 data:
-  # Configures the default for any Broker that does not specify a spec.config or Broker class.
   default-br-config: |
     clusterDefault:
       brokerClass: MTChannelBasedBroker
       apiVersion: v1
       kind: ConfigMap
-      name: config-kafka-channel
+      name: config-br-default-channel
       namespace: knative-eventing
     namespaceDefaults:
       namespace-1:
         apiVersion: v1
         kind: ConfigMap
-        name: config-br-default-channel
+        name: kafka-channel
         namespace: knative-eventing
       namespace-2:
         apiVersion: v1
         kind: ConfigMap
-        name: config-br-default-channel
+        name: kafka-channel
         namespace: knative-eventing
 ```
 
+
+## Configuring the Broker class
+
+Besides configuring the Broker class for each broker [individually](create-broker.md#broker-class-options), it is possible to define the default Broker class cluster wide or on a per namespace basis:
+
+### Configuring the default BrokerClass for the cluster
+
+You can configure the `clusterDefault` Broker class so that any Broker created in the cluster that does not have a `BrokerClass` annotation uses this default class:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+data:
+  # Configures the default for any Broker that does not specify a spec.config or Broker class.
+  default-br-config: |
+    clusterDefault:
+      brokerClass: MTChannelBasedBroker
+```
+
+### Configuring the default BrokerClass for namespaces
+
+You can modify the default Broker class for one or more namespaces.
+
+For example, if you want to use a `KafkaBroker` class for all other Brokers created on the cluster, but you want to use the `MTChannelBasedBroker` class for Brokers created in `namespace-1` and `namespace-2`, you would use the following ConfigMap settings:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+data:
+  # Configures the default for any Broker that does not specify a spec.config or Broker class.
+  default-br-config: |
+    clusterDefault:
+      brokerClass: KafkaBroker
+    namespaceDefaults:
+      namespace1:
+        brokerClass: MTChannelBasedBroker
+      namespace2:
+        brokerClass: MTChannelBasedBroker
+```
+
+  !!! note
+      Be aware that different broker classes usually require different configuration ConfigMaps. See the section at the beginning for an example on a per-namespace configuration of the used ConfigMap.
```

Comment:
Not clear enough - if referring back to another section, please provide a link. Also, this doesn't state which broker classes require different configmaps or what those configmaps would look like. I'd prefer if we open issues to document those specific configmaps and broker classes properly with examples provided.

Reply:
Added a bit more context and linked to the Broker class specific configurations. Is it clearer now?